### ISSUE TITLE
Hockey App provider for OS X app

### ISSUE THREAD

**Issue by miguelgazela** [orta/ARAnalytics#261]
I'm trying to use ARAnalytics in a OS X application with the Hockey App provider. I've successfully installed them with Cocopods, but after configuring it with

`[ARAnalytics setupWithAnalytics: @{
                                       ARHockeyAppBetaID : @"appKeyHere",
                                       }];`

if I log the current list of providers it's empty. It didn't add the provider.

Any idea of what is going on here?

Thanks in advance

**Comment by orta** [comment]
Looks like these: https://github.com/orta/ARAnalytics/blob/master/ARAnalytics.m#L380-L393 only take Hockey  iOS into account, if you add checks for `AR_HOCKEYAPPOSX_EXISTS` in those function you can make it create [the mac provider](https://github.com/orta/ARAnalytics/blob/master/Providers/HockeyAppOSXProvider.m#L4) too, looks like an oversight

**Comment by miguelgazela** [comment]
Yap, that fixed the issue, current providers shows a HockeyAppOSXProvider. However, now when I use

`[ARAnalytics event:@"something"]`

I don't see anything happening. Where should I be able to access the log for my events?

**Comment by orta** [comment]
https://github.com/orta/ARAnalytics/blob/master/Providers/HockeyAppOSXProvider.m#L69 depends on this logging stuff really, Hockey does do events per say.

**Comment by miguelgazela** [comment]
Yeah, it's not being considered compatible, so it's not really doing anything. How can that be? I installed the HockeyAppOSX using Cocoapods

`pod 'ARAnalytics', :subspecs => ['HockeyAppOSX']`

**Comment by orta** [comment]
I'm not sure, you'll have to do some research

**Comment by miguelgazela** [comment]
Ok, thanks anyway!
